src/index.js

```diff
@@ -1,87 +1,65 @@
 /* eslint-disable prefer-template */
 import './style.css';
 
-let scores = [
-  {
-    name: 'Name: ',
-    score: 127,
-  },
-];
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/X9huMz5c5d7SxuSpxECG/scores/';
 
-const setStored = () => localStorage.setItem('savedScore', JSON.stringify(scores));
-const getStored = () => JSON.parse(window.localStorage.getItem('savedScore'));
-
-class AddScore {
-  constructor(name, score) {
-    this.name = name + ': ';
-    this.score = score;
-  }
+// GET
+async function fetchScore() {
+  const response = await fetch(baseURL);
+  return response.json();
 }
 
-// DOM
-const scoreContainer = document.querySelector('.scores');
-const formContainer = document.querySelector('.form');
-
-// Form
-const ul = document.createElement('ul');
-const liOne = document.createElement('li');
-const inputOne = document.createElement('input');
-const inputTwo = document.createElement('input');
-const liTwo = document.createElement('li');
-const button = document.createElement('button');
-button.classList.add('button');
-const btnClass = document.querySelector('.button');
-const submit = document.createTextNode('Submit');
-button.appendChild(submit);
-formContainer.appendChild(ul);
-ul.appendChild(liOne);
-liOne.appendChild(inputOne);
-ul.appendChild(liTwo);
-liTwo.appendChild(inputTwo);
-formContainer.appendChild(button);
-
-const iterateScore = () => {
-  scores.forEach((item, i) => {
-    scores[i].index = i;
-    const div = document.createElement('div');
-    div.classList.add('container');
-    const name = document.createElement('p');
-    const getName = document.createTextNode(item.name);
-    name.appendChild(getName);
-    const number = document.createElement('p');
-    const getScore = document.createTextNode(item.score);
-    number.appendChild(getScore);
-    div.appendChild(name);
-    div.appendChild(number);
-    scoreContainer.appendChild(div);
+// POST
+// My game ID NVMs7bDqCFiWd9Tmg47Y: console.log(response.json())
+// First we create the ID for the object { name: 'Franks Game'}
+async function postScore(objectData) {
+  const response = await fetch(baseURL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(objectData),
   });
-};
+  console.log(response.json());
+}
 
-const clearPrevious = () => {
-  const oldList = document.querySelectorAll('.container');
-  [...oldList].forEach((e) => e.remove());
-};
+// RENDER
+const renderScore = () => {
+  fetchScore()
+    .then((serverData) => {
+      let html = '';
+      serverData.result.forEach((data) => {
+        const htmlSegment = `<p>${data.user}: ${data.score}</p>`;
```

Comment:
- [ ] Your app does not return anything after submitting and refreshing. I have checked the `API response with the browser` and it seems nothing is being returned 😢 . You can kindly follow this [article](https://support.demandbase.com/hc/en-us/articles/205180803-Checking-the-API-Response-with-your-Browser) to see how you can check the API response in your browser.